src/components/PlayerCardContainer/PlayerCard.js

```diff
@@ -17,11 +15,16 @@ function PlayerCard(props) {
         props.onBalanceChange(event, props.id)
     }
 
+    function gameHasStarted(gameStarted) {
+        props.startGame(gameStarted)
+    }
+
+
     return (
         <form className="player-card">
-            <CustomInput type="text" classes='input-name' changeHandler={nameHandler} placeHolder='Enter Name' />
-            <CustomInput type="number" classes='input-balance' changeHandler={balanceHandler} placeHolder='Enter starting balance' />
-            <CustomButton winOrLost='W' classes='winner' player={{name: props.name, balance: props.balance}}/>
+            <CustomInput type="text" classes={'input-name'} name={props.name} changeHandler={nameHandler} hasGameStarted={props.hasGameStarted} placeHolder='Enter Name' />
+            <CustomInput type="number" classes='input-balance' balance={props.balance}  changeHandler={balanceHandler} hasGameStarted={props.hasGameStarted} placeHolder='Enter starting balance' />
+            <CustomButton winOrLost='W' classes='winner' player={{name: props.name, balance: props.balance}} gameStarted={gameHasStarted} hasGameStarted={props.hasGameStarted}/>
```

Comment:
gameStarted={gameHasStarted} hasGameStarted={props.hasGameStarted}

These names are really confusing to me. Is there some way to reword this so it's more clear on the intent?  What is the difference between gameHasStarted and hasGameStarted?

Separately, I like that you have name and balance going into a player prop object for the custom button.  Is there a reason you're not passing the full player object down to this PlayerCard component?

Reply:
No reason as to why I wasn't passing just the player prop object, good catch refactored to pass just the player prop object vs each individual prop from the player. Please let me know if naming makes more sense.